src/components/Timeline/src/StepContent.tsx

```diff
@@ -35,7 +37,13 @@ export interface StepContentProps extends BaseProps {
  * @constructor Constructs an instance of StepContent.
  */
 export const StepContent: React.FC<StepContentProps> = (props: StepContentProps) => {
-  return <MaterialStepContent {...props}>{props.children}</MaterialStepContent>;
+  return (
+    <StylesProvider injectFirst>
+      <MaterialStepContent className="denhaag-timeline__step-content" {...props}>
```

Comment:
For consistencies sake I would prefer this className to also be in the bem-mapping file

Reply:
And same comment with regards to `{...props}` as in `Step.tsx`

Reply:
I was unable to do this. If I removed this MUI complained `Material-UI: <StepContent /> is only designed for use with the vertical stepper.` Even when I passed all the properties described in the documentation https://material-ui.com/api/step-content/

Reply:
Alright, nice work on checking it out anyway. Let's just leave it for now and add a issue on the backlog to change this when the next release of MUI is out.

Reply:
That still leaves my comment wrt the className, my apologies, gets a bit confusing when multiple findings are documented in one thread.